mysite/polls/models.py

```diff
@@ -17,6 +18,11 @@ def was_published_recently(self):
         now = timezone.now()
         return now >= self.pub_date >= timezone.now() \
             - datetime.timedelta(days=1)
+    was_published_recently.short_description = 'Опубликовано недавно'
```

Comment:
You should not add Russian text to code. Russian text should be in a translation file.